api/routes/utils.js

```diff
@@ -90,39 +95,74 @@ var utils = {
     return investor;
   },
 
-  // Given experts ids invested by an investor
-  // For each expert, remove investments from that investor
-  updateInvestorsExperts: function(expertIds, categoryName, investorId, cb) {
-    // If the list of experts is empty, simply return
-    if (expertIds.length === 0) {
-      return cb(null);
+  // For each investment, decrement that experts reps and remove the investor if necessary
+  // Give the investment amounts back to the investor
+  // Decrement the investment amounts from the category market size
```

Comment:
Oh, so this doesn't deal with the issue where it doesn't get removed from their investor table.